crates/uv/tests/pip_install.rs

```diff
@@ -1245,6 +1328,80 @@ fn install_git_private_https_pat_not_authorized() {
     "###);
 }
 
+/// Install a package from a private GitHub repository using a PAT
+/// Does not use `git`, instead installs a distribution artifact.
+/// Include a public GitHub repository too, to ensure that the authentication is not erroneously copied over.
+#[test]
+#[cfg(all(not(windows), feature = "git"))]
+fn install_github_artifact_private_https_pat_mixed_with_public() {
```

Comment:
As far as I can tell, these don't fail on main but seem nice to have regardless.